mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go

```diff
@@ -97,6 +98,34 @@ func LossTest(nodeToIsolate string, otherNodes []string, disconnectionMethod str
 	common.RunFio("fio", 20)
 }
 
+// Remove the replica without running IO and verify that the volume becomes degraded but is still functional
+func LossWhenIdleTest(nodeToIsolate string, otherNodes []string, disconnectionMethod string, uuid string) {
+	fmt.Printf("disconnecting \"%s\"\n", nodeToIsolate)
+
+	DisconnectNode(nodeToIsolate, otherNodes, disconnectionMethod)
+
+	fmt.Printf("waiting up to 90s for disconnection to affect the nexus\n")
+	//time.Sleep(90 * time.Second)
```

Comment:
I think we can remove the commented out line and the print statement.

Reply:
I'll shortly be having another PR, so I'll remove it then. Thanks.